client/human/HumanClientFSM.cpp

```diff
@@ -351,7 +352,8 @@ boost::statechart::result WaitingForMPJoinAck::react(const JoinGame& msg) {
             try {
                 std::string cookie_option = HumanClientApp::EncodeServerAddressOption(Client().Networking().Destination());
                 GetOptionsDB().Remove(cookie_option + ".cookie");
-                GetOptionsDB().Add(cookie_option + ".cookie", "OPTIONS_DB_SERVER_COOKIE", boost::uuids::to_string(cookie));
+                GetOptionsDB().Add(cookie_option + ".cookie", "OPTIONS_DB_SERVER_COOKIE", boost::uuids::to_string(boost::uuids::nil_uuid()));
```

Comment:
not relevant to these changes, but why `Remove` then `Add` rather than the `OptionExists` test used on the server?

Might also be clearer to `<std::string>` parameterize `Add` like on the server as well.

Reply:
As I remember `Remove`+`Add` was done to reset default value as well. Looks like it isn't issue anymore.

Reply:
Fixed